src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -1460,6 +1460,30 @@ void TableDockWidget::writeGroupXML(QXmlStreamWriter& stream, PeakGroup* g) {
         stream.writeAttribute("formula", QString(c->id.c_str()));
     }
 
```

Comment:
Add documentation for this function in the header file. And in-code documentation wherever required.